addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -381,7 +382,15 @@ def attributes(self, record, field_name, options, values=None):
         if options.get('inherit_branding'):
             field = record._fields[field_name]
             if field.sanitize:
-                attrs['data-oe-sanitize'] = 1 if field.sanitize_form else 'allow_form'
+                if field.sanitize_overridable:
+                    if not record.user_has_groups('base.group_sanitize_override'):
+                        try:
+                            field.convert_to_column(record[field_name], record)
+                        except UserError:
+                            attrs['data-oe-sanitize-prevent-edition'] = 1
+                else:
+                    attrs['data-oe-sanitize'] = 1 if field.sanitize_form else 'allow_form'
```

Comment:
This should actually be part of the first commit, only the part `not has_group('sanitize_override')` should be part of this commit. But after the first commit, it's already true that a sanitize_overridable field for someone who *has* the sanitize_override group should not be marked with `data-oe-sanitize`.

Nitpicking I guess, especially if we `rebase-merge` but even in that case... :) (btw would you prefer rebase-merge or rebase-ff ?)

Reply:
Re-read what I said ^^ I am not saying the part in "not has_group" should be in the first commit, I am saying the exact opposite :grimacing: 

> after the first commit, it's already true that a sanitize_overridable field **for someone who has the sanitize_override group** should not be marked with data-oe-sanitize

The code I comment is:
```
if sanitize_overridable
     if not has_group
          do something
     else
          do nothing
else
     add sanitize attribute
```

The kinda-hidden "do nothing" part, should be part of the first commit, the do something part should stay in the commit it is currently in.

Reply:
> which is what you did in the end

Well, if despite the misunderstanding, the code was adapted as planed, the change certainly made sense :+1: 

Note that about the PR, I believe everything from everyone has been processed so far.